execution_workflows/DaPars/Dockerfile

```diff
@@ -10,9 +10,9 @@ ENV DEBIAN_FRONTEND noninteractive
 RUN apt-get update && apt-get install -y curl unzip python python-dev python3 python3-pip r-base bedtools wget
 
 # Install DaPars
-RUN curl -LJO gitreleases.dev/gh/ZhengXia/dapars/latest/dapars_0.9.1.zip
-RUN unzip -q "dapars_0.9.1.zip"
-RUN chmod +x /dapars/src/*
+RUN wget https://github.com/ZhengXia/dapars/archive/master.zip
+RUN unzip -q "master.zip"
+RUN chmod +x /dapars-master/src/*
 ENV PATH=$PATH:/dapars/src/
```

Comment:
should be `ENV PATH=$PATH:/dapars-master/src/`
right now, it throws an `DaPars_Extract_Anno.py: command not found` error, which is because the path setting here is not correct.